pkg/controller/manager/manager_controller.go

```diff
@@ -190,12 +190,6 @@ func (r *ReconcileManager) Reconcile(request reconcile.Request) (reconcile.Resul
 	reqLogger.V(2).Info("Loaded config", "config", instance)
 	r.status.OnCRFound()
 
-	// Write the manager back to the datastore.
-	if err = r.client.Update(ctx, instance); err != nil {
```

Comment:
i'm curious how this could be removed without causing problems?

Reply:
I think this was needed when there was Auth stuff in Manager but that has now been moved to the Authentication CR. GetManager now checks that some fields are not set on Manager and reports they should be set on Authentication CR instead.